CK3toEU4/Source/CK3World/Titles/Titles.h

```diff
@@ -21,6 +21,7 @@ class Titles: commonItems::parser
 	void linkTitles();
 	void linkCharacters(const Characters& characters);
 	void linkLandedTitles(const LandedTitles& landedTitles);
+	void relinkDFVassals();
```

Comment:
```suggestion
	void relinkDeFactoVassals();
```
Really makes it a lot clearer for someone less familiar with CK.

Reply:
I hear you but entire converter uses DF and DJ as shortcuts for defacto and dejure. Otherwise names are just too long. But sure, I can fix the function name.